README.md

```diff
@@ -132,39 +146,75 @@ To calculate emissions for all layers, run `omPrior.py` with a start and end dat
+Outputs can be found in the `data/outputs` folder. The emissions layers will be written as variables to a copy of the
+input domain file, with an `OCH4_` prefix for the methane layer variable names. The sum of all layers will be stored in
+the `OCH4_TOTAL` layer.
 
 The name of the layered output file will be `om-prior-output.nc`.
 
-The `data/processed` folder will contain any re-projected raster data, and any files downloaded or generated in the process.
+The `data/processed` folder will contain any re-projected raster data, and any files downloaded or generated in the
+process.
 
 ## Layers
 
 Many sectors are taken from data sets used in by Saunois et al (2020) (doi:10.5194/essd-12-1561-2020)
 
-- Livestock: Enteric fermentation emissions generated by CSIRO Ag. and Food using livestock census data and UNFCCC emissions factors
+- Livestock: Enteric fermentation emissions generated by CSIRO Ag. and Food using livestock census data and UNFCCC
+  emissions factors
 - Electricity: Uses OpenNEM facility data to spatialise the Aust. Gov UNFCCC electricity emissions
-- Agriculture: Agricultural emissions apart from livestock taken from the Agricultural emissions of the NGGI and spatialised according to the agriculture land-use mask
+- Agriculture: Agricultural emissions apart from livestock taken from the Agricultural emissions of the NGGI and
+  spatialised according to the agriculture land-use mask
 - Fugitives: Facility-level data from ACF (more info?)
 - Industrial: Spatialises the industrial sector of the NGGI according to nighttime lights
 - Stationary: Spatialises the stationary energy sector of the NGGI according to nighttime lights
 - Transport: Spatialises the transport sector of the NGGI according to nighttime lights
 - Waste: Spatialises the NGGI waste emission according to the landuse map
 - LULUCF: Spatialises the LULUCF emission from the NGGI according to the landuse map
 - FIRE: daily emissions from the Global Fire Assimilation System (Kaiser et al., 2012, doi:10.5194/bg-9-527-2012)
-- wetland: Monthly wetland emissions from the diagnostic ensemble used in Saunois et al. 2020 and described in Zhang et al. (2023 under review)
+- wetland: Monthly wetland emissions from the diagnostic ensemble used in Saunois et al. 2020 and described in Zhang et
+  al. (2023 under review)
 - Termite: Termite emissions used in Saunois et al. 2020 supplied by Simona Castaldi and Sergio Noce
+
+## Run in a Docker container
+
+To carry out the steps described above in a Docker container, first build the Docker image with
+
+```
+make build
+```
+Then run the commands to download the input data in the docker container
+
+```
+docker run --rm -v </your/path/to/openmethane-prior>:/opt/project openmethane-prior python scripts/omDownloadInputs.py
+```
```

Comment:
Is this actually relevant for users? Or do you only need docker if you run the application on a server like we do?

Reply:
True. Good point. If we make developer docs then docker usage can go there, but for now assume that you are running locally